PyQt5-stubs/sip.pyi

```diff
@@ -44,7 +44,9 @@ T = TypeVar("T")
 
 
 # The array type.
-class array(Sequence[T]): ...
+class array(Sequence[T]):
+    def __getitem__(self, params: int) -> T: ...
```

Comment:
https://mypy-play.net/?mypy=latest&python=3.8&gist=e50679b96d7812bd36563ee5d7467ef4
```python
from typing import Any, Sequence, TypeVar


T = TypeVar("T")


class array(Sequence[T]):
    def __getitem__(self, key: Any) -> Any: ...
    def __len__(self) -> int: ...


class again(Sequence[T]):
    def __getitem__(self, key: int) -> Any: ...
    def __len__(self) -> int: ...


class more(Sequence[T]):
    def __getitem__(self, key: Any) -> T: ...
    def __len__(self) -> int: ...
```
```
main.py:13: error: Signature of "__getitem__" incompatible with supertype "Sequence"
main.py:18: error: Signature of "__getitem__" incompatible with supertype "Sequence"
Found 2 errors in 1 file (checked 1 source file)
```

ugh...  have to just `Any` both the index and the result?  What in the world is the use of this being a `Generic`?

Reply:
Naw, I'm getting somewhere better.  In progress...